package.json

```diff
@@ -77,8 +78,8 @@
     "url": "git+https://github.com/mozilla/blurts-server.git"
   },
   "scripts": {
-    "start": "npm run build & node server.js",
-    "dev": "npm run build & nodemon server.js",
+    "start": "source .env && if [[ $MONITOR_V2 == 'true' ]]; then npm start --workspace=src; else npm run build & node server.js; fi;",
+    "dev": "source .env && if [[ $MONITOR_V2 == 'true' ]]; then npm run dev --workspace=src; else nodemon server.js; fi;",
```

Comment:
This works OK in local development, and will work in Docker because `.env-dist` is copied to `.env` in the image. I think it will fail in Heroku\, where `.env` doesn't exist by default. I tried this:

```sh
> mv .env .env.back
> npm start

> blurts-server@1.0.0 start
> source .env && if [[ $MONITOR_V2 == 'true' ]]; then npm start --workspace=src; else npm run build & node server.js; fi;

/var/folders/_y/72__kkbs2tqd5ch4gwrgnwtr0000gn/T/start-204f5703.sh: line 1: .env: No such file or directory
```

On way would be to test existence first:

```suggestion
    "start": "if [[ -f .env ]]; then source .env; fi && if [[ $MONITOR_V2 == 'true' ]]; then npm start --workspace=src; else npm run build & node server.js; fi;",
    "dev": "if [[ -f .env ]]; then source .env; fi && if [[ $MONITOR_V2 == 'true' ]]; then npm run dev --workspace=src; else nodemon server.js; fi;",
```

Another way may be explicit commands:

```suggestion
    "start:v1": "npm run build & node server.js",
    "dev:v1": "npm run build & nodemon server.js",
    "start:v2": "npm start --workspace=src",
    "dev:v2": "npm run dev --workspace=src",
    "start": "if [[ $MONITOR_V2 == 'true' ]]; then npm run start:v2; else npm run start:v1; fi;",
    "dev": "if [[ $MONITOR_V2 == 'true' ]]; then npm run dev:v2; else npm run dev:v1; fi;",
```

Then v2 could be selected by environment variable `MONITOR_V2` or explictly with `npm run start:v2`.

Reply:
Thanks for flagging this!  

Testing existence in the first example seems verbose.  However, the second example could have confusing behavior locally for `npm start`,  i.e. a dev expects `npm start` to "just work", but they will always get v2 even if they set `$MONITOR_V2`.

Whatever we decide, we can probably just limit it to `npm start` since that's the command used by other environments, vs `npm run dev` which is local-only?

Agreed with getting Heroku fixed in a separate PR – in fact I had already created a Jira ticket this morning to get that going 👍